flutter-idea/src/io/flutter/run/FlutterReloadManager.java

```diff
@@ -64,6 +59,7 @@
 import io.flutter.settings.FlutterSettings;
 import io.flutter.utils.FlutterModuleUtils;
 import io.flutter.utils.MostlySilentColoredProcessHandler;
+import org.assertj.core.util.Strings;
```

Comment:
Is this a String utility?  Where is it used below?  Consider org.apache.commons.lang3.StringUtils which we use elsewhere already?